.eslintrc

```
@@ -1,3 +1,11 @@
 {
-  "extends": "gulp"
+  "extends": "gulp",
+  "rules": {
+    "node/no-unsupported-features/es-syntax": [
+      2,
+      {
+        "version": ">=10.13.0"
+      }
+    ]
+  }
```

```suggestion
  "extends": "gulp"
```
Not until gulp 6 work